src/state/local_test.go

```diff
@@ -36,9 +36,9 @@ func TestLoadLocalStateComplex(t *testing.T) {
 	state := loadLocalTestState(t, "complex")
 	assert.NotNil(t, state)
 	assert.Equal(t, 0, state.LocalErrors().Len())
-	assert.Equal(t, complexLocalExpectedBranches(), state.Branches())
-	assert.Equal(t, complexLocalExpectedConfigs(), state.Configs())
-	assert.Equal(t, complexLocalExpectedConfigRows(), state.ConfigRows())
+	assert.Equal(t, complexLocalExpectedBranches(), utils.SortByName(state.Branches()))
```

Comment:
V testoch je problem s nahodnými ID a poradim objektov. 
Pridal som metodu co pole zoradi podla mena, aby som to uz nemusel riesit na kazdom miesto osobitne.